### PR TITLE
Cisdp 1040

### PR DESCRIPTION
**Pull request by slamothe99**
## Pre-review checklist

I have ensured that:

- [ ] Code runs successfully in the development environment.
- [ ] New code is tested to prove that it works. If not, justify reasoning.
- [ ] New code is documented using [type hinting](https://realpython.com/lessons/type-hinting/) and following the [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) docstring format.
- [ ] New code follows the project naming standards.

## Post-review checklist

As I merge these changes, I have:

- [ ] Updated the related Jira ticket, including approximate time spent (including review).
- [ ] Moved the related Jira ticket to Done.

### CODE REVIEW

File: cishouseholds/pipeline/unassayed_blood_ETL.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+from cishouseholds.pipeline.validation_schema import unassayed_blood_validation_schema
+
+# from cishouseholds.derive import assign_column_uniform_value
+# from cishouseholds.derive import assign_substring
+# from cishouseholds.derive import assign_test_target
+# from cishouseholds.derive import assign_unique_id_column
+
+
+@register_pipeline_stage("unassayed_blood_ETL")
+def blood_delta_ETL(**kwargs):
```
Comment (slamothe99) [inline cishouseholds/pipeline/unassayed_blood_ETL.py:19]: unassayed_bloods_ETL --change name

File: cishouseholds/pipeline/timestamp_map.py [hunk 2]
```diff
@@ -20,3 +20,5 @@
     "yyyy-MM-dd HH:mm:ss": ["blood_sample_collected_datetime"],
     "yyyy-MM-dd": ["blood_sample_arrayed_date", "antibody_test_result_recorded_date", "blood_sample_received_date"],
 }
+
+unprocessed_blood_datetime_map = {""}
```
Comment (slamothe99) [inline cishouseholds/pipeline/timestamp_map.py:24]: remove this

File: cishouseholds/pipeline/unassayed_blood_ETL.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        df = df.select(sorted(df.columns))
+        update_table_and_log_source_files(df, "unassayed_blood_test_data", "unassayed_blood_source_file")
+
+
+def transform_unassayed_blood(df: DataFrame) -> DataFrame:
+    """
+    Call functions to process input for unassayed blood.
+    """
+    df = assign_filename_column(df, "unassayed_blood_source_file")
+    # df = assign_test_target(df, "antibody_test_target", "unassayed_blood_source_file")
```
Comment (foster999) [inline cishouseholds/pipeline/unassayed_blood_ETL.py:38]: Delete unused calls

File: cishouseholds/pipeline/timestamp_map.py (hunk 2 above)
Comment (foster999) [inline cishouseholds/pipeline/timestamp_map.py:24]: Delete this dict if unused

File: cishouseholds/pipeline/unassayed_blood_ETL.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    if file_path_list:
+        df = extract_validate_transform_input_data(
+            file_path_list,
+            unassayed_bloods_variable_name_map,
+            blood_datetime_map,
+            unassayed_blood_validation_schema,
+            transform_unassayed_blood,
+        )
+        df = df.select(sorted(df.columns))
+        update_table_and_log_source_files(df, "unassayed_blood_test_data", "unassayed_blood_source_file")
```
Comment (foster999) [inline cishouseholds/pipeline/unassayed_blood_ETL.py:30]: Please could "test" be removed from the table name:
```suggestion
        update_table_and_log_source_files(df, "unassayed_blood_sample_data", "unassayed_blood_source_file")
```

**Review by foster999 — CHANGES_REQUESTED** [review]

**Review by foster999 — APPROVED** [review]
Review changes actioned

**Pull request merged**